tox.ini

```diff
@@ -83,6 +83,7 @@ setenv =
     dev: ENABLE_WEB = {env:ENABLE_WEB:true}
     dev: ENABLE_WEBSOCKET = {env:ENABLE_WEBSOCKET:true}
     dev: ENABLE_WORKER = {env:ENABLE_WORKER:true}
+    dev: ALEMBIC_CONFIG = {env:ALEMBIC_CONFIG:conf/alembic.ini}
```

Comment:
Needed to be able to run alembic commands without setting it all the time.

This is now present it most other projects.